client/src/layouts/BaseLayout.scss

```
@@ -1,10 +1,26 @@
 @import 'styles/variables';
+@import 'styles/helpers';
+
+body {
+  background-color: $color-primary;
```

I find this wall of red a bit "aggressive" to the eye. Maybe tone it down by using a lighter variant ?
```suggestion
  background-color: lighten($color-primary, 20%);
```